sprocs/sync_question_tags.sql

```diff
@@ -0,0 +1,35 @@
+-- Accepts a course ID and a list of objects containins qids and tag IDs,
```

Comment:
I don’t see the course ID being passed here?

Reply:
copy/paste gone wrong 🙂